src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/ChoiceList/ChoiceList.spec.jsx

```diff
@@ -53,4 +53,18 @@ describe('Text component', () => {
 
         expect(cmp.props().value).toBe('Yolooo');
     });
+
+    it('should be readOnly when formDefinition say so', () => {
```

Comment:
Can you make two different it block ?

Reply:
I guess in this case, it's not important, but If you think so, okay